samples/SimpleWebAPISample/Program.cs

```diff
@@ -1,39 +1,50 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Linq;
-using System.Threading.Tasks;
 using Microsoft.AspNetCore;
 using Microsoft.AspNetCore.Hosting;
 using Microsoft.Extensions.Configuration;
-using Microsoft.Extensions.Logging;
-
 using Serilog;  
 
 namespace SimpleWebAPISample
 {
     public class Program
     {
-        public static void Main(string[] args)
+        public static int Main(string[] args)
         {
+            var configuration = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
```

Comment:
One slight annoyance here - any idea how to get the information about the hosting environment - Development, Production etc. - without resolving `IHostingEnvironment` from DI?